prisma/schema.prisma

```diff
@@ -10,24 +10,98 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Exhibitor {
-  id                 String  @id @default(uuid())
+model ExhibitorInterestRegistration {
+  id                 String   @id @default(uuid())
+  createdAt          DateTime @default(now())
   name               String
-  organizationNumber String?
+  organizationNumber String
   contactPerson      String
   phoneNumber        String
+  email              String
+
+  @@map("exhibitor_interest_registrations")
+}
+
+model Exhibitor {
+  id                 String @id @default(uuid())
+  name               String
+  organizationNumber String
+
+  invoiceEmail              String
+  logoWhite                 Bytes?
+  logoColor                 Bytes?
+  description               String
```

Comment:
Do we not want to make the description optional? Unless not having one is an empty string (i.e. "") which also makes sense.

Reply:
I think an empty one is better, since that avoids having two "null" states